thirdparty/runtime_environments.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	Packages   = "Packages"
```

Comment:
nit: could we change the variable name to be bit specific 
ie `EnvTypePackage` or something like that